Bio/GenBank/__init__.py

```diff
@@ -466,10 +466,10 @@ def __init__(self, debug_level=0, use_fuzziness=1,
     def parse(self, handle):
         """Parse the specified handle.
         """
-        self._consumer = _FeatureConsumer(self.use_fuzziness,
-                                          self._cleaner)
-        self._scanner.feed(handle, self._consumer)
-        return self._consumer.data
+        _consumer = _FeatureConsumer(self.use_fuzziness,
```

Comment:
Unit tests still passed, so doesn't seem to be affecting much internally. If other people were accessing it, maybe there will be an angry email? We could also just set a null self._consumer in the __init__.